shape/ttmath/ttmathtypes.h

```diff
@@ -241,7 +241,7 @@ namespace ttmath
 		/*!
 			on 64bit platforms one word (uint, sint) will be equal 64bits
 		*/
-		typedef unsigned long uint;
+		typedef unsigned long long uint;
```

Comment:
@lixun910 I should pay attention to this change:  

unsigned long long is guaranteed to be at least 64 bits, regardless of the platform.  unsigned long is guaranteed to be at least 32 bits.

Reply:
Anyway, this could be discarded by switching to boost::centroid